lib/hyrax/errors.rb

```diff
@@ -0,0 +1,14 @@
+module Hyrax
+  # Generic Hyrax exception class.
+  class HyraxError < StandardError
+  end
+
+  # Error that is raised when an active workflow can't be found
+  class MissingWorkflowError < HyraxError
+  end
+
+  class WorkflowAuthorizationException < HyraxError
+  end
+
+  class SingleUseError < StandardError; end
```

Comment:
Should this inherit from `HyraxError` like the others? Also, should we break the `end` onto the next line like the others?